### ISSUE TITLE
Grab content type from the spec first

### ISSUE THREAD

**Issue by amritk** [scalar/scalar#3319]
### What happens?

We use the dropdown to grab the content type, but we should instead grab it from the spec and only if the dropdown is changed, update it

### What did you expect to happen?

We should load the correct content-type from the spec

**Comment by hanspagel** [comment]
For the response body?

**Comment by amritk** [comment]
requestbody: so currently if its `application/something+json` it doesn't automatically get added

**Comment by amritk** [comment]
Alright we got it sorted, different content types in the request body will create different examples

**Comment by marclave** [comment]
closing since fixed :)

**Comment by amritk** [comment]
I don't think we actually fixed it @marclave ! or did we? I would have linked this ticket in the PR if so

**Comment by JamesLawton** [comment]
Yep can confirm this is still an issue: https://discord.com/channels/1135330207960678410/1298223830979969064/1303317990502301726

**Comment by amritk** [comment]
I will take a look at this one soon time permitting 🙏🏾

**Comment by hanspagel** [comment]
This works way better than it did, so I’m closing this here. :)
